docs/data/joy/components/stepper/stepper.md

```diff
@@ -0,0 +1,107 @@
+---
+productId: material-ui
```

Comment:
A quick fix d2cd42d6540d6612cc71cb5ddffec7a967c993e8. This was flagged in the SEO crawl. Bug: https://deploy-preview-39688--material-ui.netlify.app/joy-ui/react-stepper/#api